chatbot/admin.py

```
@@ -0,0 +1,303 @@
+            'fields': ('id', 'user', 'session_type', 'status')
+        }),
+        ('زمان‌بندی', {
+            'fields': ('started_at', 'last_activity', 'ended_at', 'expires_at', 'duration_display')
+        }),
+        ('داده‌ها', {
+            'fields': ('context_data', 'metadata'),
+            'classes': ('collapse',)
+        }),
+    )
+    
+    def conversation_count(self, obj):
```

Ruff (0.12.2)

Missing return type annotation for public function `conversation_count`

(ANN201)